bentoml/_internal/log.py

```diff
@@ -94,10 +95,24 @@ def configure_logging():
     logging.config.dictConfig(CLI_LOGGING_CONFIG)
 
 
+@lru_cache(maxsize=1)
+def _component_name():
+    result: str = (
+        component_context.component_type
+        if component_context.component_type is not None
+        else ""
+    )
```

Comment:
imo separating these into two lines are better for better clearance.

Reply:
I'm not exactly sure if I understood the suggestion. Would you like to add a suggestion and I will commit.